ubuntu20.04/val-verde-platform-sdk-baikonur

```diff
@@ -29,7 +29,7 @@ ${PACKAGE_BASE_NAME}-platform-sdk-swift-build
 cd ${STAGE_ROOT}
 mkdir -p install${BUILD_PACKAGE_PREFIX}/bin
 mkdir -p install${BUILD_PACKAGE_PREFIX}/lib/swift/${SDK_PLATFORM_DIR}
-rsync -aPx release/vapor* \
+rsync -aPx release/baikonur* \
            install${BUILD_PACKAGE_PREFIX}/bin
 rsync -aPx release/*.swiftdoc \
```

Comment:
These next two rsyncs are not needed. Only the baikonur binary is installed, rest are intermediates and can be discarded.

Reply:
removed